asmcli/components/ca/managed-cas.sh

```diff
@@ -114,7 +114,7 @@ x_wait_for_enabling_workload_certificates() {
   local REACHABLE; REACHABLE=0
   for i in {1..3}; do
     local OUT; OUT=$(kubectl get ds -n kube-system gke-spiffe-node-agent 2>/dev/null | grep -i "gke-spiffe-node-agent")
-    if [[ -z "$OUT" ]]; then
+    if [[ -n "$OUT" ]]; then
```

Comment:
Why changes -z to -n here?

Reply:
IIUC, this condition should check if OUT is not nil. That is accomplished with the -n flag? the -z flag is the opposite?

Reply:
The -z flag checks the string is empty or not, whereas -n checks a variable is nil or not. A non-nil variable may have an empty string. So "-z" should be used here.

Reply:
okay with leaving it as -z, but should the conditions be reversed then? if [[ -z ${OUT} ]]; then means cluster should be unreachable right?